### PR TITLE
Quick Documentation Fix for Sliver

### LINKED ISSUE #108619: Typo in documentation for SliverChildListDelegate
There is a missing backtick before `children` in this doc comment.

```
  /// If the order of children` never changes, consider using the constant
  /// [SliverChildListDelegate.fixed] constructor.
```

### PR DESCRIPTION
**Pull request by youssef-attia**
Small fix in documentation for SliverChildListDelegate

#108619 

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/sliver.dart [hunk 1]
```diff
@@ -571,7 +571,7 @@ class SliverChildListDelegate extends SliverChildDelegate {
   /// [addSemanticIndexes], and [semanticIndexCallback] arguments must not be
   /// null.
   ///
-  /// If the order of children` never changes, consider using the constant
+  /// If the order of children never changes, consider using the constant
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/sliver.dart:574]: Nit:

```suggestion
  /// If the order of children never changes, consider using the constant.
```

**Comment by Piinks** [comment]
Hey @youssef-attia the flutter_plugins failure here is not internal, it is from flutter/plugins here on github. The failure does not have to do with your PR, but you will need to update you branch here with the latest from master for it to pass.

File: packages/flutter/lib/src/widgets/sliver.dart (hunk 1 above)
Comment (youssef-attia) [reply packages/flutter/lib/src/widgets/sliver.dart:574]: I think the rest of the sentence is on the next line.

Comment (Piinks) [reply packages/flutter/lib/src/widgets/sliver.dart:574]: Oh! The github UI cut it off. lol 🙃

**Review by Piinks — APPROVED** [review]
Flutter_LGTM

**Pull request merged**